web/urls.ts

```diff
@@ -20,4 +20,6 @@ export const urls = {
 
   kiosk: (action_id: string): `/kiosk/${string}` => `/kiosk/${action_id}`,
   team: (): "/team" => "/team",
+
+  debugger: () => "/debugger",
```

Comment:
@paolodamico One small thing, can we add explicit return type here? Improves the displaying in IDE with typescript: `() => "/debugger"` instead of `() => string`